packages/hint/docs/contributor-guide/guides/create-custom-hint.md

```diff
@@ -24,9 +24,15 @@ a resource file.
 The recommended way to create a new hint is to use `npm create hint` from
 the CLI.
 
-**Note for core hints**: If you are creating a hint for the main repo you should
-use the same command from the `packages` folder. The generated files will be
-slighthly different (references and task will change a bit).
+>**Note for core hints**: If you are creating a hint for the main repo you should
+>follow the below steps:
+>
+>* Make sure to run `yarn` and `yarn build` in root directory.
+>* Then go to `create-hint` package ( `cd packages/create-hint` ).
+>* Make executable release. run `yarn webpack`.
+>* Now run `npm create hint` from `packages` folder.
+>
+>The generated files will be slighthly different (references and task will change a bit).
```

Comment:
The length of this line is causing the PR to fail during markdown linting in CI. Just need to wrap it to reduce the length below 80 characters.

Reply:
Oh ok. Thanks. I am fixing it.

Reply:
done